.gitignore

```diff
@@ -127,3 +133,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+>>>>>>> main
```

Comment:
Can you merge main from upstream to resolve the merge conflict. Otherwise looks great